scripts/aws/deploy_studio.py

```diff
@@ -35,16 +35,16 @@
                         help="Location of landing page for 'root' level requests")
     parser.add_argument("--edxapp-host", required=True,
                         help="Location of edxapp for request routing")
-    parser.add_argument('--studio-host', required=True,
-                        help="Location of Studio IDA for request routing")
+    parser.add_argument('--cms-host', required=True,
```

Comment:
Is there any benefit to doing an expand/contract with the two names here, or is it fine for the caller to just brake until it is updated?

Reply:
I'll generate a parallel PR in the `edx-internal` repo to make the changes here be non-breaking relative to GoCD, and then cross-link the two PRs. The `edx-internal` change can be merged first, as merges to `api-manager` are very infrequent, and merging of this PR is virtually guaranteed to be the next time this repo gets merged.

Reply:
Actually, I've gone the KISS route, and removed `cms-host` for now. `edxapp-host` serves just as well. If at some point in the future the two diverge, we can deal with it then.